app/src/server/utils/copyDatasetEvalDatasetEntries.ts

```diff
@@ -0,0 +1,19 @@
+import { prisma } from "../db";
+
+export const copyDatasetEvalDatasetEntries = async (
```

Comment:
Will this always be used in the same place we call `updatePruningRules`, which are the same places we're creating updated dataset entries?

I think we should create a helper `updateDatasetEntry` that takes in an entry id as well as an object of fields to update, then does the update and calls these as needed. Too easy to forget a place otherwise, like I did with `output_format` (and I don't think I ever fixed, actually).

Reply:
I think that's a great idea. I'd like to do that in a separate PR, since there's already a lot of refactoring going on in this one.